lib/core-common/package.json

```diff
@@ -76,6 +76,7 @@
     "find-up": "^5.0.0",
     "fork-ts-checker-webpack-plugin": "^6.0.4",
     "glob": "^7.1.6",
+    "glob-base": "^0.3.0",
```

Comment:
why is this needed? i think we tried to get rid of it here: #15399

Reply:
Good catch!